sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -511,6 +516,8 @@ def __init__(self, **kwargs):
         self.encryption_key_sha256 = kwargs.get('x-ms-encryption-key-sha256')
         self.encryption_scope = kwargs.get('x-ms-encryption-scope')
         self.request_server_encrypted = kwargs.get('x-ms-server-encrypted')
+        self.object_replication_source_properties = kwargs.get('object_replication_source_properties')
+        self.object_replication_destination_policy = kwargs.get('x-ms-or-policy-id')
```

Comment:
we also want to parse the headers in _from_generated if list_blobs returns the replication headers

Reply:
We synced offline, this will be added as part of a different PBI.